azurelinuxagent/common/protocol/extensions_config_retriever.py

```diff
@@ -0,0 +1,405 @@
+# Microsoft Azure Linux Agent
```

Comment:
For every module <path>/<module>.py we create, we have a corresponding tests/<path>/test_<module>.py with the unit tests for that module.

Could you do that for extensions_config_retriever.py?

Reply:
Added test_extensions_config_retriever.py